### PR TITLE
soc/sifive/fu540: Update QEMU detect logic

### PR DESCRIPTION
**Pull request by alistair23**
There was another recent QEMU RISC-V ROM change, so let's update Oreboot
to handle the different values.

Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

### CODE REVIEW

File: src/soc/sifive/fu540/src/lib.rs
Comment (fotonick) [inline src/soc/sifive/fu540/src/lib.rs:17]: Please update comment to match.

**Comment by alistair23** [comment]
Also now we are finally using a released version of QEMU for the CI!

**Comment by alistair23** [comment]
I rebased and dismissed the review. Can I get a second review then I'll merge it?

**Comment by alistair23** [comment]
Ping

**Review by fotonick — APPROVED** [review]

**Pull request merged**
